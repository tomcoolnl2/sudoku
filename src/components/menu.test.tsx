
import { render } from '@testing-library/react'
import { withTheme } from '../test'
import { withRouter } from '../test'
import { Menu } from '.'

describe('Menu', () => {
	
	it('should match it\'s snapshot', () => {
		const { container } = render(withTheme(withRouter(<Menu />)))
		expect(container.firstChild).toMatchSnapshot()
	})
})