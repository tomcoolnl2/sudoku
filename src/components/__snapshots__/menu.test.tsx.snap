// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match it's snapshot 1`] = `
<ul>
  <li>
    <a
      href="/"
    >
      Start
    </a>
  </li>
  <li>
    <a
      href="/settings"
    >
      Settings
    </a>
  </li>
</ul>
`;
